utils/release/npm-publish-package.mjs

```diff
@@ -44,6 +47,22 @@ const isPrerelease = process.env.IS_PRERELEASE === 'true';
 
 // Run on each package, it generate the changelog, install the latest dependencies that are part of the workspace, publish the package.
 (async () => {
+  // Lock-out master
+  await changeMasterWriteAccess(false);
+
+  // Verify master has not changed
+  const local = await getSHA1fromRef('master');
+  await gitPull();
+  const remote = await getSHA1fromRef('master');
+  if (!process.env.NO_LOCK && local !== remote) {
```

Comment:
wouldn't it be easier to read without the double negative?

Reply:
We can use another name for the variable name, but the `!process.env.OTHER_NAME` should stay:
Indeed, we're testing the presence of the variable, because it is not set outside of the E2E tests.